README.md

```diff
@@ -1,17 +1,45 @@
 
 ![SuperTokens banner](https://raw.githubusercontent.com/supertokens/supertokens-logo/master/images/Artboard%20%E2%80%93%2027%402x.png)
 
-# SuperTokens Website SDK
+# SuperTokens Javascript Frontend SDK
 
 <a href="https://supertokens.io/discord">
 <img src="https://img.shields.io/discord/603466164219281420.svg?logo=discord"
     alt="chat on Discord"></a>
-    
-## About
-This is a website / JavaScript SDK that is responsible for maintaining a SuperTokens session for website apps (SPA and non-SPA).
+
+SuperTokens adds **secure login and session management** to your apps. This is the Javascript Frontend SDK for SuperTokens. [More SDKs are available](https://github.com/supertokens) for frontend and backend e.g. Node.js, Go, Python, React.js, React Native, Vanilla JS, etc.
+
+![Architecture Diagram](https://supertokens.com/img/architecture/self_hosted_generic.png)
+SuperTokens architecture is optimized to add secure authentication for your users without compromising on user and developer experience
+
 
 Learn more at https://supertokens.io
 
+
+# How to install
+
+```
+<script src="https://cdn.jsdelivr.net/gh/supertokens/supertokens-website/bundle/bundle.js"></script>
+```
+
+# Example
```

Comment:
```suggestion
# Initialising
```

Reply:
I'm not sure, why?

Reply:
Cause an example would be making an request. The contents of this section is about initialising the SDK

Reply:
New changes should convey this as well as keep the heading simple